src/main/java/net/dv8tion/jda/core/utils/WidgetUtil.java

```diff
@@ -191,33 +190,44 @@ public static Widget getWidget(String guildId) throws RateLimitedException
      */
     public static Widget getWidget(long guildId) throws RateLimitedException
     {
-        Args.notNull(guildId, "GuildId");
+        Checks.notNull(guildId, "GuildId");
         try
         {
-            HttpResponse<JsonNode> result = Unirest.get(String.format(WIDGET_URL, guildId)).asJson();
-            switch(result.getStatus())
+            HttpURLConnection connection = (HttpURLConnection) new URL(String.format(WIDGET_URL, guildId)).openConnection();
+            connection.setRequestMethod("GET");
+            connection.addRequestProperty("user-agent", Requester.USER_AGENT);
+            connection.connect();
+            
+            switch(connection.getResponseCode())
             {
-                case 200: return new Widget(result.getBody().getObject()); // ok
+                case 200: // ok
+                    try (InputStream stream = connection.getInputStream())
+                    {
+                        return new Widget(new JSONObject(new JSONTokener(stream)));
+                    }
+                    catch (IOException ignored) {
+                        return null;
+                    }
```

Comment:
Add a `{}` scope to the case here, looks a bit weird imo and the other case has it.